snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/core/domain/SnomedDescription.java

```diff
@@ -227,6 +229,29 @@ public String getCaseSignificanceId() {
 		return acceptabilities;
 	}
 	
+	/**
+	 * Return the common description equivalent of the description type identifier
+	 * 
+	 * @see Description
+	 * @return common description type
+	 */
+	public Description.Type getCommonDescriptionType() {
```

Comment:
There are multiple issues with this method:
* Introduces a new JSON property in the serialized form of this representation class
* Returns UNKNOWN for synonym subtypes
* Returns UNKNOWN for partially loaded descriptions
Let's chat, why do you need this new enum type and where are you going to use it.